CodeGeneration/Sources/SyntaxSupport/KeywordSpec.swift

```diff
@@ -97,6 +97,7 @@ public let KEYWORDS: [KeywordSpec] = [
   KeywordSpec("availability"),
   KeywordSpec("available"),
   KeywordSpec("await"),
+  KeywordSpec("backDeployed"),
```

Comment:
Should we remove the underscored `_backDeploy` now?

Reply:
We have to continue to parse the old spelling since it's in use. I'm not sure whether it needs to be in this list or not to support that though.

Reply:
Ah, sorry, I missed that you still supported the old spelling. Makes sense to me 👍